server/src/model/init.ts

```diff
@@ -16,6 +16,7 @@ const addInitialBoardSchema = async () => {
   schema.addString("voting", {defaultValue: "disabled"});
   schema.addNumber("votingIteration", {required: true, defaultValue: 0});
   schema.addBoolean("showVotesOfOtherUsers", {defaultValue: false});
+  schema.addBoolean("showNotesOfOtherUsers", {defaultValue: true});
```

Comment:
I think the already existing `showContentOfOtherUsers` is the flag you are looking for

Reply:
Yes. I will remove this flag as it is no longer used.